components/claim/components/email_to_oauth.tsx

```diff
@@ -12,26 +11,34 @@ import {t} from 'utils/i18n.jsx';
 import LoginMfa from 'components/login/login_mfa.jsx';
 import LocalizedInput from 'components/localized_input/localized_input';
 
-export default class EmailToOAuth extends React.PureComponent {
-    static propTypes = {
-        newType: PropTypes.string,
-        email: PropTypes.string,
-        siteName: PropTypes.string,
-    };
+export type State = {
+    showMfa?: boolean;
+    password?: string;
+    error?: string;
```

Comment:
Could these values be made non-nullable if we specify values for them in the constructor? Since they all have default zero values like the empty string or false, I think we can keep things simpler by using those instead of null/undefined.

Reply:
How do I change the value to empty string or false?

Reply:
you'll need to initialize it with a value like `error=''` so that it at least contains the required type